vtysh/vtysh.c

```diff
@@ -1896,7 +1896,7 @@ DEFUN (vtysh_show_debugging,
        DEBUG_STR)
 {
 	return show_per_daemon("do show debugging\n",
-			       "Debugging Information for %s:\n");
+			       "");
```

Comment:
no \n

Reply:
I don't want a \n  I want no output at all like what it is doing here.